oidc-service-impl/src/main/java/io/mosip/esignet/services/LinkedAuthorizationServiceImpl.java

```diff
@@ -110,7 +117,8 @@ public LinkCodeResponse generateLinkCode(LinkCodeRequest linkCodeRequest) throws
         linkCodeResponse.setLinkCode(linkCode);
         linkCodeResponse.setTransactionId(linkCodeRequest.getTransactionId());
         linkCodeResponse.setExpireDateTime(expireDateTime == null ? null :
-                expireDateTime.format(DateTimeFormatter.ofPattern(UTC_DATETIME_PATTERN)));
+                expireDateTime.format(DateTimeFormatter.ofPattern(UTC_DATETIME_PATTERN)));        
+        auditWrapper.logAudit(Action.LINK_CODE, ActionStatus.SUCCESS, AuditHelper.buildAuditDto(linkCodeRequest.getTransactionId(), null), null);
```

Comment:
we have the OIDC transaction object available, then why are we still passing it as null to AuditHelper.buildAuditDto ?

Reply:
Passing transaction to buildAuditDto